feat: Add `airport-codes` package

gflujan opened the pull request:
This is a collection of airport code abbreviations that expand into the city names where they reside.

This is a port of the same expansion set found in TextExpander.

cl3mcg commented:
Hello @gflujan,
I tried to submit an espanso package that is also related to airport codes.
Do you want to work on a common package ? I don't really know if it is possible, but if you want, let me know.
https://github.com/espanso/hub/pull/92

gflujan commented on packages/airport-codes/0.1.0/_pkgsource.yml:
Probably not. 🙂 I can remove it.

smeech replied:
Thank you. I think we're good to go. Just awaiting someone else to review, in case I've overlooked something, and then it'll be merged,

gflujan commented:
> Hello @gflujan, I tried to submit an espanso package that is also related to airport codes. Do you want to work on a common package ? I don't really know if it is possible, but if you want, let me know. #92

Hi @cl3mcg, thanks for the offer! But as @smeech noted, I think our two packages are solving for different use cases. 

But, I'd be happy to add a link to your package in my README if you're willing to do the same. 🙂

cl3mcg commented:
> > Hello @gflujan, I tried to submit an espanso package that is also related to airport codes. Do you want to work on a common package ? I don't really know if it is possible, but if you want, let me know. #92
> 
> Hi @cl3mcg, thanks for the offer! But as @smeech noted, I think our two packages are solving for different use cases.
> 
> But, I'd be happy to add a link to your package in my README if you're willing to do the same. 🙂

Hello @gflujan and thank you for your reply.
[I have updated the README.md](https://github.com/espanso/hub/pull/92/commits/8e20a296d4a183d790c2f0b76ff57acc2fa771be) of my pull request to include a reference and link to your package. 😄
Have a nice day.

gflujan commented:
> > > Hello @gflujan, I tried to submit an espanso package that is also related to airport codes. Do you want to work on a common package ? I don't really know if it is possible, but if you want, let me know. #92
> > 
> > 
> > Hi @cl3mcg, thanks for the offer! But as @smeech noted, I think our two packages are solving for different use cases.
> > But, I'd be happy to add a link to your package in my README if you're willing to do the same. 🙂
> 
> Hello @gflujan and thank you for your reply. [I have updated the README.md](https://github.com/espanso/hub/pull/92/commits/8e20a296d4a183d790c2f0b76ff57acc2fa771be) of my pull request to include a reference and link to your package. 😄 Have a nice day.

Awesome, thanks @cl3mcg! I've updated [my README](https://github.com/gflujan/espanso-hub/blob/main/packages/airport-codes/0.1.0/README.md) as well to have a link to your package. 🙂
